assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/index.js

```diff
@@ -296,7 +305,9 @@ const SearchFunnelWidget = ( {
 		analyticsStatsData === undefined ||
 		analyticsVisitorsOverviewData === undefined ||
 		analyticsVisitorsStatsData === undefined ||
-		analyticsGoalsData === undefined
+		analyticsGoalsData === undefined ||
+		isAnalyticsGatheringData === undefined ||
+		isSearchConsoleGatheringData === undefined
 	) {
 		return (
 			<Widget Header={ Header } Footer={ WidgetFooter } noPadding>
```

Comment:
I can't comment on it here but the main problem this issue was opened for is further below in the logic that shows the full-widget gathering data message:
https://github.com/google/site-kit-wp/blob/ca22b6b3709bf49213e786a55babe7e0589db79a/assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/index.js#L331-L337

This needs to be updated to use the gathering data value rather than zero data as mentioned in the ACs.

This also simplifies the changes in `Overview` as that component will only ever be rendered if SC is not gathering data, so those selectors and conditions can be removed. Similarly, checks for SC zero data can be removed because we don't show anything differently in this case any more if the module isn't gathering data (which again is guaranteed here). This was somewhat hidden by non-zero data that was being received in the zero data story for SC here.